kafka/src/main/java/org/jboss/aerogear/unifiedpush/kafka/streams/NotificationRouterStreamsHook.java

```diff
@@ -71,22 +77,19 @@
 
     private final String STREAMS_INPUT_TOPIC = KafkaClusterConfig.NOTIFICATION_ROUTER_STREAMS_INPUT_TOPIC;
 
-    public final String ADM_TOPIC = "agpush_admPushMessageTopic";
+    private final String ADM_TOPIC = "agpush_admPushMessageTopic";
```

Comment:
for CDI bean these need to be private, since public fields can not be proxied.
 See: https://access.redhat.com/solutions/463743